### PR TITLE
Get reviews by

### PR DESCRIPTION
**Pull request by scarlett-k-nhs**

### CODE REVIEW

File: app.test.js [hunk 1]
```diff
@@ -46,4 +46,41 @@ describe('getCategories', () => {
...
+})
+
+describe('getReviewsById', () => {
+  test("Responds with a review object with correct properties", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({body}) => {
+        expect(body.review).toMatchObject({
+          review_id: expect.any(Number),
```
Comment (Slyose) [inline app.test.js:58]: Also make sure that the array length is  > 0 in this test, as empty arrays can sneak by otherwise.

File: app.test.js [hunk 2]
```diff
@@ -46,4 +46,41 @@ describe('getCategories', () => {
...
+          title: expect.any(String),
+          designer: expect.any(String),
```
Comment (Slyose) [inline app.test.js:92]: There should also be a review_body property on this object.

File: app.test.js (hunk 1 above)
Comment (Slyose) [inline app.test.js]: You can expect the review_id here to be 1, because of your endpoint.

**Review by Slyose — APPROVED** [review]
A few changes to be made, then hit merge!

File: app.test.js (hunk 1 above)
Comment (scarlett-k-nhs) [inline app.test.js]: But the test only asks for an object not an array
